app/src/controllers/user/constants.js

```diff
@@ -1,12 +1,14 @@
 export const FETCH_USER_SUCCESS = 'fetchUserSuccess';
 export const SET_ACTIVE_PROJECT = 'setActiveProject';
 export const SET_START_TIME_FORMAT = 'setStartTimeFormat';
+export const SET_PHOTO_TIME_STAMP = 'setPhotoTimeStamp';
 
 export const START_TIME_FORMAT_RELATIVE = 'relative';
 export const START_TIME_FORMAT_ABSOLUTE = 'absolute';
 
 export const SETTINGS_INITIAL_STATE = {
   startTimeFormat: START_TIME_FORMAT_RELATIVE,
+  photoTimeStamp: null,
```

Comment:
Maybe it should be initalized as `Date.noew()`?

Reply:
Fixed